api/gradlew

```diff
@@ -32,10 +32,10 @@
 #       Busybox and similar reduced shells will NOT work, because this script
 #       requires all of these POSIX shell features:
 #         * functions;
-#         * expansions �$var�, �${var}�, �${var:-default}�, �${var+SET}�,
-#           �${var#prefix}�, �${var%suffix}�, and �$( cmd )�;
-#         * compound commands having a testable exit status, especially �case�;
-#         * various built-in commands including �command�, �set�, and �ulimit�.
+#         * expansions «$var», «${var}», «${var:-default}», «${var+SET}»,
```

Comment:
Stimmt hier die Codierung?

Reply:
Ich habe das File nicht von Hand angepasst, war ein Command über CLI um Gradle auf 7.5 hochzuheben. Gemäss IntelliJ ist es mit Line Ending `LF` und `UTF-8`. Da dies sowieso nur in den Kommentaren ist, sehe ich kein grosses Problem.